.pre-commit-config.yaml

```diff
@@ -4,6 +4,9 @@ repos:
     hooks:
```

Comment:
Good that black now follows pep8 conventions.

Reply:
I totes agree